File: 8kyu/(8 kyu) Total amount of points/(8 kyu) Total amount of points.ts
export function points(games: string[]): number {
  return games.reduce((res, game) => {
    const [x, y] = game.split(":");

    if (+x > +y) {
      return (res += 3);
    } else if (x === y) {
      return (res += 1);
    }

    return res;
  }, 0);
}

File: 8kyu/(8 kyu) Total amount of points/(8 kyu) Total amount of points.cpp
int points(const std::array<std::string, 10>& games) {
  int res = 0;
  for (auto game: games) {
    int x = game[0];
    int y = game[2];
    if (x > y) {
        res += 3;
    } else if (x == y) {
        res += 1;
    }
  }
  return res;
}